### PR TITLE
#19982 method to update uuid of containers

### PR DESCRIPTION
**Pull request by erickgonzalez**
We need to set the UUID of each container that is in a template, to accomplish this we need to iterate over each container in the template and assign a UUID according to the number of times the container is present in the template.

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateLayoutView.java [hunk 1]
```diff
@@ -50,4 +52,39 @@ public BodyView getBody() {
...
+                    containerUUID -> {
+                        final String containerID = containerUUID.getIdentifier();
+                        int value = UUIDByContainermap.getOrDefault(containerID, 0);
+                        UUIDByContainermap.put(containerID, ++value);
+                        containerUUID.setUuid(UUIDByContainermap.get(containerID).toString());
+                    }
+            );
+        }
+
+        return this;
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateLayoutView.java:88]: Returning doesn't seem to be needed

**Review by nollymar — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateLayoutView.java [hunk 2]
```diff
@@ -50,4 +52,39 @@ public BodyView getBody() {
...
+
+    /**
+     * This method iterates over each container in the template and assign a UUID according the
+     * amount of times the container is present in the template.
+     * This is for the users can have 2 diff templates with diff design but the same containers
+     * so they can change the template without losing the content.
+     *
+     * @return TemplateLayoutView with the updated UUID for each container.
+     */
+    protected TemplateLayoutView updateUUIDOfContainers(){
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateLayoutView.java:64]: Looks like it can be void

**Review by freddyucv — CHANGES_REQUESTED** [review]
any test?

**Comment by dsilvam** [comment]
> any test?

Test will be sent to `release-21.03` @freddyucv

**Review by dsilvam — APPROVED** [review]

**Pull request merged**